tools/pywebsocket/example/echo_client.py

```diff
@@ -51,6 +51,7 @@
 # run echo client to test IETF HyBi 00 protocol
  run with --protocol-version=hybi00
 """
+from __future__ import print_function
```

Comment:
Same question @gsnedders , but at least here upstream seems dead.

Reply:
This is in `tools/` and not `tools/third_party/` because we've already forked it, so this is fine from my POV.